types/react-native/index.d.ts

```diff
@@ -7396,12 +7411,7 @@ export interface AppStateStatic {
      * Add a handler to AppState changes by listening to the change event
      * type and providing the handler
      */
-    addEventListener(type: AppStateEvent, listener: (state: AppStateStatus) => void): void;
-
-    /**
-     * Remove a handler by passing the change event type and the handler
-     */
-    removeEventListener(type: AppStateEvent, listener: (state: AppStateStatus) => void): void;
```

Comment:
Instead of removing, can we mark it as `@deprecated` just like you did [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55354/files#diff-e94ad9b79ba4f07235bfefa12c2789cd2d6cb690d9120309555f33ae51295244R7709)?

Reply:
I totally missed this comment last week. I was under the impression the method was removed, but you're right. It looks like it was restored in a later PR. https://github.com/facebook/react-native/commit/035718ba97bb44c68f2a4ccdd95e537e3d28690c#diff-3402d8d0082acba7a28c141bf8ca9aa3ae63c4dd9dfcf210c3d4b5bea739bdc1

I'd normally make this change right now, but am on vacation this week, and am not sure I will be able to quickly follow up once sign off happens again. It's been a couple of weeks since 0.65 has been out, so I think it would be better to cut the typings as-is to unblock adoption, with 0.66 coming very soon.

I'm going to merge this change as-is, but am keeping this in mind as a followup for when I am back. Folks have been great on this thread in finding fixups, and I am hoping merging what is here now as 0.65 should unblock folks PR'ing them directly.